tvm-disasm-cli/src/main/kotlin/org/ton/PrettyPrint.kt

```diff
@@ -0,0 +1,45 @@
+package org.ton
+
+import org.ton.bytecode.*
```

Comment:
Please, do not use star imports (if they are automatically generated, you can turn this off in your IDE settings)